db/migrate/20211230221517_create_products.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProducts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :products do |t|
+      t.string :name
+      t.string :code
+      t.float :price
```

Comment:
Probably you are using float instead of double because you are using the SQLite.